src/slices/AccountSlice.ts

```diff
@@ -80,6 +85,8 @@ export const getBalances = createAsyncThunk(
   async ({ address, networkID, provider }: IBaseAddressAsyncThunk): Promise<IUserBalances> => {
     let gOhmBalance = BigNumber.from("0");
     let gOhmBalAsSohmBal = BigNumber.from("0");
+    let gOhmOnEthereum = BigNumber.from("0");
+    let gOhmOnEthAsSohm = BigNumber.from("0");
```

Comment:
are these 2 groups duplicates of each other ?

Reply:
`gOhmBalance` is using the user connected wallet as provider, this means, if you'r connected to avax it will pull your gOhm balance on the avax chain, the `gOhmOnEthereum` is using the eth rpc to ensure it's eth. 

It's fetching twice if your wallet is connected to one of the supported chains, I'm working on a fix